core/sqf/sql/scripts/sqstart

```diff
@@ -220,6 +220,31 @@ function SQCheckOrphanProcesses {
 
 }
 
+function checkKerberos {
+
+   # Check to see if kerberos is enabled in Hadoop
+   cat /etc/hadoop/conf/core-site.xml | while read a; do
+     found=`echo $a | grep "hadoop.security.authentication" | wc -l`
+     if [[ $found -eq 1 ]]; then
+       read b
+       enabled=`echo $b | grep kerberos | wc -l`
```

Comment:
This code seems to assume that kerberos is on the very next line after "hadoop.security.authorization". Which is probably true but not necessarily. They could be on the same line, for example, or a comment line could occur between them. That said, without a full-blown XML parsing this is probably as good as you'll get.